changelog/13549.bugfix.rst

```diff
@@ -0,0 +1 @@
+Pytest no longer evaluates type annotations on Python 3.14 when inspecting a function signature.
```

Comment:
The correct spelling is "pytest" (not "Pytest"), which makes it awkward to start sentences with it, so reword a bit to side step the issue:

```suggestion
No longer evaluate type annotations in Python ``3.14`` when inspecting function signatures. 

This prevents crashes during module collection when modules do not explicitly use ``from __future__ import annotations`` and import types for annotations within a ``if TYPE_CHECKING:`` block.
```